docs/hub/_toctree.yml

```diff
@@ -4,401 +4,403 @@
   title: Repositories
```

Comment:
maybe try to keep the changes to this file as minimal as possible to avoid merge conflicts ? ^^